src/interval.jl

```diff
@@ -228,26 +231,26 @@ end
 Base.:⊈(a::AbstractInterval, b::AbstractInterval) = !issubset(a, b)
 Base.:⊉(a::AbstractInterval, b::AbstractInterval) = !issubset(b, a)
 
-function overlaps(a::AbstractInterval, b::AbstractInterval)
+function isoverlapping(a::AbstractInterval, b::AbstractInterval)
     left = max(LeftEndpoint(a), LeftEndpoint(b))
     right = min(RightEndpoint(a), RightEndpoint(b))
 
     return left <= right
 end
 
-function contiguous(a::AbstractInterval, b::AbstractInterval)
+function iscontiguous(a::AbstractInterval, b::AbstractInterval)
```

Comment:
I think `is` implies a predicate with one value. I think `contiguous` is fine but I don't have an issue with changing this. If you do change it, however, it should be `arecontiguous`.

Reply:
Based upon [Alex's comment](https://github.com/invenia/Intervals.jl/pull/42/commits/89216aa06a69a344d8d8de68ff2811278eda6260#r203103948) should this be `contiguouswith`? For example `a contiguouswith b`.

Reply:
Alternatively: `adjacentto`

Reply:
If we were to go with the "adjacent" terminology rather than "contiguous," I think `adjacent` itself is a fine enough name. That said, I don't really have a problem with `contiguous`, but I find `continugouswith` to be a bit of a mouthful.